ethcore/src/snapshot/tests/service.rs

```diff
@@ -103,6 +102,9 @@ fn restored_is_equivalent() {
 
 #[test]
 fn guards_delete_folders() {
+	let gas_prices = vec![1.into(), 2.into(), 3.into(), 999.into()];
+	let client = generate_dummy_client_with_spec_and_data(Spec::new_null, 400, 5, &gas_prices);
```

Comment:
iirc, this helper function generates client with 400 blocks. you should check in this test if they were actually migrated correctly

Reply:
I'm not sure I understand. This test only tests if the guarded folders are deleted.

Reply:
from #6350

> When restoring from a snapshot parity should try and re-import ancient blocks existing in the current database starting from genesis.

We need a test for that ;)

Reply:
Ah yeah sure, I thought you were commenting on the `guards_delete_folders` test!